Makefile

```diff
@@ -25,6 +25,8 @@ install-deps:
 # Go
 	go get github.com/Masterminds/glide
 	go get github.com/GeertJohan/go.rice/rice
+	go get github.com/kardianos/osext
+	go get github.com/google/gopacket
 	glide install
```

Comment:
Hi,
Le 12 avr. 2016 1:10 AM, Jason Ish <notifications@github.com> a écrit :
>
> In Makefile:
>
> > @@ -25,6 +25,8 @@ install-deps:
>
> >  # Go
>
> >  	go get github.com/Masterminds/glide
>
> >  	go get github.com/GeertJohan/go.rice/rice
>
> > +	go get github.com/kardianos/osext
>
> > +	go get github.com/google/gopacket
>
> >  	glide install
>
> Did glide install not download these 2 for you?
No, I had to install them manually, multiple run of install-deps were not enough.
++
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub